app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Safe to assume that any files under `app/views/devise` are auto-generated by Devise (Ruby authentication dependency) unless otherwise specified. In fact, all are auto-generated, the only thing to note is that the views for the "Sign Up" and "Log In" page are slightly modified to look nicer and use some of the Bootstrap markup and styles.